play-json/shared/src/main/scala/play/api/libs/json/Reads.scala

```diff
@@ -386,11 +381,7 @@ trait DefaultReads extends LowPriorityDefaultReads {
    */
   implicit val javaBigDecReads = Reads[java.math.BigDecimal](js =>
```

Comment:
I don't see the benefit there of having `parseBigDecimal` instead of just `javaBigDecReads` and composing from there to implement `bigDecReads`

Reply:
Updated